docs/UserGuide.adoc

```diff
@@ -126,75 +132,127 @@ Format: `change password o/YOUR_OLD_PASSWORD n/YOUR_NEW_PASSWORD`
+TIP: All commands pertaining to contacts start with either `client` or `vendor`.
 
-Format: `client add n/FULL_NAME p/PHONE_NUMBER e/EMAIL_ADDRESS a/HOME_ADDRESS [t/TAG]...`
+==== Adding a contact
+
+You can `add` a contact together with its particulars into Heart² by specifying its contact type first, that is, `client` or `vendor`.
+
+Format: `<CONTACT_TYPE> add n/FULL_NAME p/PHONE_NUMBER e/EMAIL_ADDRESS a/HOME_ADDRESS [t/TAG]…​`
 
 [cols="^,<5a", frame=none]
 |=====
 |image:exampleimage.png[width="64", role="center"]
 |`client add n/John Doe p/87654321 e/johndoe@gmail.com a/123 Lorem Street, #45-67, Singapore 890123`
 |=====
+[cols="^,<5a", frame=none]
+|=====
+|image:exampleimage.png[width="64", role="center"]
+|`vendor add n/Mary Jane p/98765432 e/maryjane@gmail.com a/123 Lorem Street, #67-89, Singapore 890321`
+|=====
 
 // tag::listClient[]
-==== Listing and searching for clients
+==== Listing and searching for contacts
 
-You can view clients in the database as a list using the `client list` command.
+You can also `list` your different contacts by specifying the contact type to be shown.
 
-If you do not key in any parameters, it will list out all the clients,
-otherwise it will list all clients matching all of the search parameters you provided.
+[cols="^,<5a", frame=none]
+|=====
+|image:exampleimage.png[width="64", role="center"]
+|`client list`
+|=====
+[cols="^,<5a", frame=none]
+|=====
+|image:exampleimage.png[width="64", role="center"]
+|`vendor list`
+|=====
+
+Heart² also supports searching via name, phone number, email, address and tags for you to quickly find your contacts.
+To search, simply append your search parameters to the back of the original command.
 
-Format: `client list [n/FULL_NAME] [p/PHONE_NUMBER] [e/EMAIL_ADDRESS] [a/HOME_ADDRESS] [t/TAG]...`
+Format: `<CONTACT_TYPE> list [n/FULL_NAME] [p/PHONE_NUMBER] [e/EMAIL_ADDRESS] [a/HOME_ADDRESS] [t/TAG]...`
 
 [cols="^,<5a", frame=none]
 |=====
 |image:exampleimage.png[width="64", role="center"]
-|* `client list`
- * `client list n/John Doe`
+|`client list n/John`
+|=====
+[cols="^,<5a", frame=none]
+|=====
+|image:exampleimage.png[width="64", role="center"]
+|`vendor list p/98765432`
 |=====
 // end::listClient[]
 
-=== Manipulating a specific client: `client#id`
+=== Manipulating a specific contact
+
+Contacts in Heart² are assigned a unique ID each for you to refer back to them.
+This unique ID allows you to refer the contact at any point of time when using Heart².
+
+[TIP]
+Commands pertaining to one specific contact have an additional unique ID appended to the back of `client` or `vendor`
+
+[TIP]
+These IDs are persistent for one session. Each restart of Heart reassigns IDs to contacts, effectively compacting the IDs of your contacts.
+
+[NOTE]
+IDs can be similar for `client` and `vendor`.
+However, since the contact type and ID comes together, they would still refer to different contacts!
 
-==== Viewing a client
+==== Viewing a contact
```

Comment:
Are you going to do the refactoring? If not you might want to change this to "Selecting a contact" haha

Reply:
I will change to view :)